frontend/src/pages/modelCatalog/const.ts

```diff
@@ -1,4 +1,4 @@
-export type ModelDetailsRouteParams = {
+export type CatalogModelCustomProps = {
```

Comment:
Just curious why the rename here? These are still used for route params and aren't driven by MR customProperties so the name could be confusing

Reply:
i just figured that they are catalog model related props, and now they are being used also for display and not just as route params

Reply:
Yeah I think maybe `CatalogModelDetailsParams` fits both use cases